### DISCUSSION TITLE
How to get the "SearXNG API base URL" ?

### DISCUSSION THREAD

**Discussion by MarkapToGo** [searxng/searxng#3758] [Q&A] [answered]
I need the SearXNG API base URL to use it for another application. Where can I find the API base URL?

**Comment by virtadpt** [comment, marked as answer]
The base URL is wherever SearxNG is being served out of.  If you just have it running locally it'll be something like http://127.0.0.1:8888/

If you're running a public(-ish) copy someplace, it'll be something like https://searxNG.example.com/

The API stuff all hangs off of the end of that.
